example_dags/example_transform.py

```diff
@@ -43,12 +43,12 @@ def last_five_animations(input_table: Table):
 
 # [START transform_example_3]
 @aql.transform
-def union_top_and_last(first_table: Table, second_table: Table):
+def union_top_and_last(first_table: Table, second_table: Table):  # skipcq: PYL-W0613
```

Comment:
`# skipcq: PYL-W0613` can be removed. I feel @kaxil  has added it globally.

Reply:
@sunank200 mentioned before, this check was failing 3h ago in this PR :(